docs/version-1.4.0.md

```diff
@@ -0,0 +1,78 @@
+# Version 1.4.0
+
+**Release Name**: CTK 1.4.0
+
+**Upgrade From**: CTK 1.3.0
+
+**Support**: Stable Release
+
+**Release Date**: 15th April, 2024
+
+## Overview
+
+The 1.4.0 version of CTK includes the following new features:
+
+**New Features in CTK 1.4.0 release** 
+1. Quality Assessment Report is now available for Quality Assessment Collections.
+   This report can be submitted for review just like Compliance Collection report. 
```

Comment:
This report is eligible for review, similar to the Compliance Collection report.